spring-cloud-stream-app-starters-docs/src/main/asciidoc/overview.adoc

```diff
@@ -1,7 +1,7 @@
 [[overview]]
 
 This section will provide you with a detailed overview of Spring Cloud Stream Application Starters, their purpose, and how to use them.
-It assumes familiarity with general Spring Cloud Stream concepts, which can be found in the Spring Cloud Stream http://docs.spring.io/spring-cloud-stream/docs/current-SNAPSHOT/reference/htmlsingle/[reference documentation]].
+It assumes familiarity with general Spring Cloud Stream concepts, which can be found in the Spring Cloud Stream http://docs.spring.io/spring-cloud-stream/docs/current-SNAPSHOT/reference/htmlsingle/[reference documentation].
```

Comment:
we should parameterize this as a variable so it is easy to check and change before a release, as in https://github.com/spring-cloud/spring-cloud-dataflow-server-cloudfoundry/blob/master/spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/index.adoc